components/shortcuts/shortcuts_modal/shortcuts_modal.test.tsx

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper';
+
+import ShortcutsModal from './shortcuts_modal';
+
+describe('components/shortcuts/ShortcutsModal', () => {
+    test('should match snapshot', () => {
+        const wrapper = mountWithIntl(
+            <ShortcutsModal/>,
```

Comment:
we should probably test if the shortcuts are rendered correctly too

Reply:
Sounds cool, but I'm not that experienced with writing tests, so someone else may have to look into this.